tests/kokkos-based/hermitian_matrix_rank1_update_kokkos.cpp

```diff
@@ -0,0 +1,262 @@
+
```

Comment:
Would you consider adding a copyright header for new files?

Reply:
Fixed